data_generators/generators.py

```diff
@@ -82,8 +82,8 @@ class RandomMeeting(Printable):
     def __init__(self, mentee_id, mentor_id):
         self.meeting_id = generate_uuid(16)
         self.meeting_topic = choice(topics)
-        self.meeting_start_time = datetime.now().isoformat()
-        self.meeting_end_time = datetime.now().isoformat()
+        self.meeting_start_time = random_datetime(datetime(2022, 1, 1), datetime(2022, 12, 31))
+        self.meeting_end_time = self.meeting_start_time + timedelta(hours=1)
         self.mentor_id = mentor_id
```

Comment:
Overall everything is consistent with the last pushes, so I'm confident it still satisfies previous approvals while avoiding the potential merge conflict that cropped up yesterday. Inline at 76 and 85-86, the origingal functionality of a random date within 2022 has been restored. Looks great.

Reply:
It was mentioned that a further Stretch Goal could be randomizing the length of the meetings themselves (so they're not all exactly 1 hour long,) but I only mention that for thorough record-keeping. In terms of this ticket, this PR, this is a full unit of work and we needn't chase the dragon of scope creep off the cliff, to mix a few metaphors.